_currencies.md

```diff
@@ -1,5 +1,117 @@
+      "precision": 8
+    },
+    "name": "Platinum",
+    "status": "open",
+    "symbol": "oz",
+    "type": "commodity"
+  },
+  {
+    "code": "AAPL",
+    "formatting": {
+      "decimal": ".",
+      "format": "__value__ __code__",
+      "grouping": ",",
+      "precision": 6
+    },
+    "name": "Apple",
+    "status": "extended",
+    "type": "equity"
+  }
+]
+```
+
+> Example of authenticated request to list only the assets available to the current user:
+
+```bash
+curl https://api.uphold.com/v0/assets \
+  -H "Authorization: Bearer <token>"
+```
+
+Get the list of supported currencies and other financial assets, with details as described by the table below:
+
+Property   | Description
+---------- | --------------------------------------------------------------------------------------------------------
+code       | Uppercase abbreviation of the asset, e.g. "BTC", USD", "C", or "BRK.B".
+formatting | Specification for user-facing display, including number formatting and placement of the code and symbol.
+name       | Full name of the asset, e.g. "Euro", "Basic Attention Token", or "0x".
+status     | Current trading status. See [below](#asset-status) for more details.
+symbol     | A short and well-known representation of the asset, if one exists — e.g. "$", "฿", or "Kč".
+type       | Type of asset — one of `commodity`, `equity`, `cryptocurrency`, `fiat`, `stablecoin`, or `utility_token`.
+
+If the request is unauthenticated, the full list of assets supported by Uphold is returned.
+Authenticated requests, on the other hand, will filter the output, returning only the assets available for the current user, which can depend on factors such as their country and state of residency.
+
+Please note that this endpoint is [paginated](#pagination), due to the large number of supported assets (hundreds), so multiple requests may be required to get the complete list.
+
+### Asset status
+
+The `status` field of an asset indicates whether it can be transacted at the moment.
+The possible values are:
+
+* `open` — the asset is in regular operation and transactions can be made with it.
+  Most assets remain in this status save for exceptional circumstances, where they can become `halted`.
+  Equities, on the other hand, will regularly transition between `open` and either `closed` or `extended`,
+  depending on their availability for 24/7 operation. See below for details on these statuses.
+* `closed` — the asset is temporarily disabled for trading;
+  used for equities during periods outside the U.S. stock market's trading hours
+  ([typically](https://www.nasdaq.com/stock-market-trading-hours-for-nasdaq) from 9:30am to 4:00pm ET on weekdays).
+* `extended` — used for equities that we make available for trading outside the U.S. stock market's trading hours.
+  Transactions made with assets in this status will settle instantly for the user,
+  but will have a higher spread, as Uphold takes financial risk to provide this service.
```

Comment:
Not sure that, from the business perspective, we want to say this. Should we validate this with someone from business?

Reply:
Thanks for the careful review :) This phrasing has been validated by Product, and is also the language we use [in the FAQ](https://support.uphold.com/hc/en-us/articles/360049468231-Fractional-Equities-24-7), so I think we're good.